_includes/install_instructions/openrefine.html

```diff
@@ -0,0 +1,51 @@
+<div id="openrefine">
+  <h3>OpenRefine</h3>
+  <p>
+    For this lesson you will need <em>OpenRefine</em> and a
+    web browser. <em>Note:</em> this is a Java program that runs on your machine (not in the cloud).
+    It runs inside a web browser, but no web connection is needed.
+  </p>
+
+  <div>
+    <ul class="nav nav-tabs nav-justified" role="tablist">
+      <li role="presentation" class="active"><a data-os="windows" href="#openrefine-windows" aria-controls="Windows" role="tab" data-toggle="tab">Windows</a></li>
+      <li role="presentation"><a data-os="macos" href="#openrefine-macos" aria-controls="MacOS" role="tab" data-toggle="tab">MacOS</a></li>
+      <li role="presentation"><a data-os="linux" href="#openrefine-linux" aria-controls="Linux" role="tab" data-toggle="tab">Linux</a></li>
+    </ul>
+
+    <div class="tab-content">
+      <article role="tabpanel" class="tab-pane active" id="openrefine-windows">
+        <p>
+          Check that you have either the Firefox or the Chrome browser installed and set as your default browser.
+          <strong>OpenRefine runs in your default browser.</strong>
+          It will not run correctly in Internet Explorer.
+        </p>
+        <p>Download software from <a href="http://openrefine.org/">http://openrefine.org/</a></p>
+        <p>Create a new directory called OpenRefine.</p>
```

Comment:
All of these `<p>`s should be changed to `<li>`. And `<ol>.... </ol>` added around them.